sys/console/minimal/src/console.c

```diff
@@ -222,6 +229,13 @@ console_handle_char(uint8_t byte)
             nlip_state |= NLIP_DATA_START1;
             break;
         case '\r':
```

Comment:
/* Falls through. */ comment is needed here as GCC7 gives an error without an explicit commented fallthrough.